webapp/static/interop-data.json

```diff
@@ -322,119 +322,118 @@
                 "description": "Viewport Units",
                 "mdn": "",
                 "spec": "https://drafts.csswg.org/css-values/#viewport-relative-units",
-                "tests": "/results/css/css-values?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2022-viewport", 
+                "tests": "/results/css/css-values?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2022-viewport",
                 "countsTowardScore": false
             },
             "interop-2022-webcompat": {
                 "description": "Web Compat",
                 "mdn": "",
                 "spec": "",
-                "tests": "/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2022-webcompat", 
+                "tests": "/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2022-webcompat",
                 "countsTowardScore": false
             },
             "interop-2023-cssborderimage": {
                 "description": "Border Image",
-                "mdn": "",
-                "spec": "",
-                "tests": "/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-cssborderimage", 
+                "mdn": "https://developer.mozilla.org/en-US/docs/Web/CSS/border-image",
+                "spec": "https://w3c.github.io/csswg-drafts/css-backgrounds/#the-border-image",
+                "tests": "/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-cssborderimage",
                 "countsTowardScore": true
             },
             "interop-2023-color": {
                 "description": "Color Spaces and Functions",
-                "mdn": "",
-                "spec": "",
+                "mdn": "https://developer.mozilla.org/en-US/docs/Web/CSS/color_value",
+                "spec": "https://w3c.github.io/csswg-drafts/css-color/#color-syntax",
                 "tests": "/results/?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2022-color%20or%20label%3Ainterop-2023-color", 
                 "countsTowardScore": true
             },
             "interop-2023-container": {
                 "description": "Container Queries",
-                "mdn": "",
-                "spec": "",
-                "tests": "/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-container", 
+                "mdn": "https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Container_Queries",
+                "spec": "https://www.w3.org/TR/css-contain-3/#container-queries",
```

Comment:
Can we link to https://drafts.csswg.org/css-contain-3/#container-queries ?

Reply:
sure